opt4j-benchmarks/src/main/java/org/opt4j/benchmarks/dtlz/DTLZModule.java

```diff
@@ -40,14 +37,14 @@
 import org.opt4j.core.start.Constant;
 
 /**
- * Module for the DTLZ benchmarks as proposed in "Scalable multi-objective
- * optimization test problems, Kalyanmoy Deb,Lothar Thiele, Marco Laumanns,
- * Eckart Zitzler, Proc. Congress Evolutionary Computation Volume 1 (May 2002)".
+ * Module for the DTLZ benchmarks as proposed in "Scalable multi-objective optimization test problems, Kalyanmoy Deb,
+ * Lothar Thiele, Marco Laumanns, Eckart Zitzler, Proc. Congress Evolutionary Computation Volume 1 (May 2002)".
  * 
  * @author lukasiewycz
  * 
  */
 @Info("DTLZ Problem Suite.  (n = m + k - 1)")
+@Citation(title = "Scalable multi-objective optimization test problems", authors = "Kalyanmoy Deb, Lothar Thiele, Marco Laumanns, and Eckart Zitzler", journal = "Proceedings of the 2002 Congress on Evolutionary Computation", volume = 1, month = MAY, year = 2002, doi = "10.1109/CEC.2002.1007032")

```

Comment:
Check given names

Reply:
I did and found no problem. What is wrong?

Reply:
See overall comment: I suggest to stick to one naming convention. It seems the majority preferred J. Doe over Jane Doe so I marked all occurrences of the latter.